modules/Microsoft.Management/managementGroups/deploy.bicep

```diff
@@ -28,14 +28,19 @@ resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (ena
   }
 }
 
+resource parentManagementGroup 'Microsoft.Management/managementGroups@2021-04-01' existing = {
+  name: parentId
```

Comment:
Since the default value of `parentId` is empty, I guess we should either change the default value to `last(split(managementGroup().id` or add a condition to this block?

Reply:
done & retested